### ISSUE TITLE
Viewing markup changes markup

### ISSUE THREAD

**Issue by bingomanatee** [tiff/wysihtml5#394]
When you view the HTML of this markup

``` html
<h1>headline</h1>
<p>
this is body</p>
<ul>
<li>bullet</li>
<li><p>bullet 2</p>
<p><img alt="dave2.jpg" src="/blog_image/dave2.jpg" title="Image:/blog_image/dave2.jpg"></p>
</li>
</ul>
<h2>Head 2</h2>
```

the relative link in the image src (in fact the whole src attribute) is erased.

Relative image src linking is very legal HTML; what do I need to do do allow it?

Thanks.

**Comment by crysfel** [comment]
Same issue here, here's my toolbar button:

```
<a data-wysihtml5-command="insertHTML" data-wysihtml5-command-value="<pre class='prettyprint'>//Code here</pre>" class="editor-button">
    <i class="icon-cogs"></i> Code
</a>
```

Clicking the button successfully add my custom pre tag, but when I switch to the HTML view the class is removed from the markup.

**Comment by kenken17** [comment]
Yes. Looks like I have the same issue. I have a button data-wysihtml5-action="change_view", when toggle with it, markup got stripped. Any idea?

**Comment by ichuan** [comment]
if you are using the latest version (currently "wysihtml5-0.4.0pre.js") and `wysihtml5ParserRules`, this can be fixed by adding this before initializing the editor:

``` javascript
wysihtml5ParserRules.tags.img.check_attributes.src = 'src';
```

The original cause I found is that, when parsing `src` attribute of `IMG` element, it used `_isLoadedImage()` to check if the image is fully loaded. But it's unpredicable for Chrome ( setting `innerHTML` of an element to a string containing `IMG` element and check `img.complete` immediately).
